### PR TITLE
Modify Regular Expression for the support of suffix with latest/fast

### PR DESCRIPTION
**Pull request by rpancham**
What this PR does / why we need it:
Automate ODH Release Process for Kserve repos

Which issue(s) this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #[RHOAIENG-2732](https://issues.redhat.com/browse/RHOAIENG-2732)

### CODE REVIEW

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
               
       - name: Update params.env with new release version
         run: |     
-          sed -i 's|:v[0-9.]*\b|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env
+          sed -i 's|:v[0-9.]*.*|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env
```
Comment (dtrifiro) [inline .github/workflows/create-release.yml:69]: Looking at `params.env`, it seems like that the  lines are in the format `name=image_name:fast`, which this regex will not match. Is there something else I'm missing?

Comment (rpancham) [reply .github/workflows/create-release.yml:69]: No , Yes, it doesn't start with v , On the release branches we will be having the tag starting with v but for the first release we will be having it has `fast` or `latest `so yesterday it was decided that for the tag we will be prefixing the version.

cc : @israel-hdez

Comment (israel-hdez) [reply .github/workflows/create-release.yml:69]: My understanding is that this new workflow would only be ran against `release-*` branches, not in `main` branch.
So, as part of our branch cut, we should modify `params.env` file to use `v{something}` tags. Ex: https://github.com/opendatahub-io/modelmesh-serving/blob/release-0.11.1/config/overlays/odh/params.env. Thus, the regex would match when the workflow is ran on `release-*` branches, which seems to be what we need.

Basically, what @rpancham said is right.
